workspaces/website/src/pages/content/@slug/SocialShare/SocialShare.tsx

```diff
@@ -0,0 +1,80 @@
+import { Flex, Icon, Text } from "@chakra-ui/react";
+import {
+  // FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton,
+  TelegramShareButton,
+} from "react-share";
+import { FaTelegram } from "react-icons/fa";
+import { FaTwitter } from "react-icons/fa";
+// import { FaDiscord } from "react-icons/fa6";
+import { FaLinkedin } from "react-icons/fa";
+
+interface Props {
+  readonly params: LocaleParams & {
+    readonly slug: string;
+  };
+  readonly env?: {
+    readonly ALGOLIA_INDEX: string;
+    readonly ALGOLIA_APP_ID: string;
+    readonly ALGOLIA_SEARCH_API_KEY: string;
+    readonly SITE_URL: string;
+  };
+}
+const SocialShare = ({ params: { slug, locale }, env }: Props) => {
+  const shareUrl = `${
+    env?.SITE_URL || "https://www.starknet.io"
```

Comment:
1. Other places in the code seem to use `import.meta.env.VITE_SITE_URL` for site URL
2. Why do you think we need ` || "https://www.starknet.io"` ?